### ISSUE TITLE
.dcm header

### ISSUE THREAD

**Issue by steel-ne** [qeda/qeda#19]
in last kicad sources I've found such definition:

https://github.com/KiCad/kicad-source-mirror/blob/master/eeschema/class_library.h
line 74

```
/* Must be the first line of part library document (.dcm) files. */
#define DOCFILE_IDENT     "EESchema-DOCLIB  Version 2.0"
```

with two spaces between DOCLIB and Version

current .dcm cause a error parsing it:
![image](https://cloud.githubusercontent.com/assets/195610/19438763/1da25ef4-9484-11e6-869b-f6c18160ca23.png)

ading space fix this error

**Comment by steel-ne** [comment]
every .dcm file from kicad distributive has two spaces

**Comment by shamil-ru** [comment]
Thank you for info. Fixed in v0.2.7.
